docs/resources/alert_rule.md

```diff
@@ -39,16 +39,15 @@ resource "valtix_alert_rule" "rule3" {
   type          = "Type_Inventory"
   sub_type      = "SubType_InventoryGuardRails"
   severity      = "Info"
-  is_active     = true
+  is_active     = "true"
```

Comment:
would this work? I thought boolean values must always be without quotes. can you please confirm